vp/devel/python_peripherals/DLA.py

```diff
@@ -193,6 +193,22 @@ def flat_to_CWH(data, channels, width, height):
                 i = i + 1
     return output
 
+def cast_long_to_signed_byte(value):
```

Comment:
I recommend adding type ascriptions in the signature for param and for return.

By personal preference, I'd name this something like `u32_into_i8` but use whatever is clearest to you.

Reply:
So as far as I understand IronPython documentation, long doesn't equate to u32, but rather Python's infinite integer signified with L (e.g 42L). i8 makes more sense here than signed byte though.